src/index.tsx

```diff
@@ -17,7 +18,12 @@ export function App() {
     <LocationProvider>
       <Header />
       <main class="neumo hollow">
-        <Router>
+        <Router
+          onRouteChange={() => {
+            // TODO may need another router lib so we can hook to the right time
```

Comment:
hook to the right time? what does this mean? Are you talking about pass in a variable of how long will the transition effect plays?

Reply:
The router library does not provide a timing for to insert the view transition properly (we need to catch the time before the routed page is going to change). So, now we only get page transition after the page is rendered (only the appearing transition but not the disappearing one). Some other router library might do, or maybe this one would be updated later to support it. However, I do not think that is a priority now and the transition is already looks okay for now.